purpose-and-process/_index.md

```diff
@@ -38,6 +38,31 @@ several projects—and their authority stems from the extent to which they are.
 Participants in the SPEC process must adhere to our
 [Code of Conduct](https://scientific-python.org/code_of_conduct/).
 
+## What is a SPEC?
+
+A SPEC (Scientific Python Ecosystem Coordination) is a document that captures an idea.
+A SPEC is the product of discussions with developers across the ecosystem.
+SPECs capture the following types of narratives:
+
+- We recommend that you do Y (e.g., [SPEC 8 — Securing the Release Process](https://scientific-python.org/specs/spec-0008/)).
+- Some projects may need to do Y. If you do Y, we recommend that you do it as follows (e.g., [SPEC 1 — Lazy Loading](https://scientific-python.org/specs/spec-0001/)).
+- If you do Y, you should be aware of the following (we don't have any such advisories yet).
```

Comment:
- is this list meant to be exhaustive?
- might it be helpful to name the types/flavors of SPECs? For the three types shown here, I'd probably call them "hortative", "advisory", and "informative".  If there are more types, TBD if sufficiently distinct words can be found.

Reply:
48a7fbbc6667688aa002fcd94020a35839b38d59

Reply:
sorry to bikeshed, but I'm pretty lukewarm on "recommendation, guideline, advisory", because:

- 2 are nouns, 1 is adjective
- "recommendation" and "advice" seem to me really similar in terms of strength (though the adjective form (advisory) does feel a bit stronger).

Here are some more ideas; pick which you like best (or stick with what you have) and I won't bring it up again. If we're going with nouns (strongest to weakest):

- exhortation, guideline, suggestion

If we're going with adjectives (and ruling out "hortative" 😅):

- advisory, guiding, informative

Reply:
ok how about:
1. recommendation
2. guideline
3. memorandum? note?

Reply:
Yes, let's go with that for now; happy to iterate as well if you awake middle of the night with a brilliant new categorization!